plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -4,8 +4,8 @@
    
    <extension point="org.eclipse.wst.common.project.facet.core.facets">        
     <project-facet id="com.google.cloud.tools.eclipse.appengine.facet">
```

Comment:
My question is somewhat related: what are the trade-offs of having separate facets for _flex_ vs _standard_ as compared to having a single facet with two different versions?  The facet versions can be strings, like _Flex_ and _Standard_, and we can provide a comparator too.